README.md

```diff
@@ -434,17 +434,38 @@ bundle exec cucumber features/your_file.feature     # Runs all scenarios in your
 bundle exec cucumber features/your_file.feature:23  # Runs scenario at line 23
 ```
 
-Or just run the whole test suite:
-
 ```
 bundle exec rake
 ```
 
+This will run the tests (RSpec and Cucumber), RuboCop and Reek itself.
+
+You can also run:
+
+```
+bundle exec rake ci
+```
+
+This will run everything the default task runs including [Ataru](https://github.com/CodePadawans/ataru) and [Mutant](https://github.com/mbj/mutant). This is the task that we run on travis as well and that determines if your pull request is green or red.
```

Comment:
I would say 'plus' or 'and also', instead of 'including'.

Also, Travis is with a capital 't'.